jobs/pull-kubernetes-federation-e2e-gce.sh

```diff
@@ -20,52 +20,45 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-# TODO(fejta): remove this
-if [[ "${PULL_BASE_REF:-}" == "release-1.0" || "${PULL_BASE_REF:-}" == "release-1.1" ]]; then
-  echo "PR GCE job disabled for legacy branches."
-  exit
-fi
-
 export FEDERATION="true"
-export PROJECT="${PROJECT:-k8s-jkns-pr-bldr-e2e-gce-fdrtn}"
-export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-pr-bldr-e2e-gce-fdrtn"
-export GINKGO_PARALLEL="n" # We don't have namespaces yet in federation apiserver, so we need to serialize
-export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Federation\]"
-export E2E_ZONES="us-central1-a us-central1-f" # Where the clusters will be created. Federation components are now deployed to the last one.
-export KUBE_GCE_ZONE="us-central1-f" #TODO(colhom): This should be generalized out to plural case
+export PROJECT="${PROJECT:--k8s-jkns-pr-bldr-e2e-gce-fdrtn}"
```

Comment:
is it intentional to use double hyphen?
PROJECT will be set to "-k8s-jkns-pr-bldr-e2e-gce-fdrtn" if not defined earlier.